xonsh/aliases.py

```diff
@@ -423,9 +423,32 @@ def detect_xip_alias():
         return basecmd
 
 
+def alias(args, stdin=None):
```

Comment:
Probably should change the signature here to add `stdout=None, stderr=None` so that...